src/generate-changelog.ts

```diff
@@ -0,0 +1,197 @@
+import {graphql} from '@octokit/graphql'
```

Comment:
@gagata @charlespierce this is the only file I'd recommend actually reviewing, the rest of the files are really just deleting a bunch of stale code after forking from GH's typescript action template